app/controllers/v0/in_progress_forms_controller.rb

```diff
@@ -5,7 +5,9 @@ class InProgressFormsController < ApplicationController
     include IgnoreNotFound
 
     def index
-      render json: InProgressForm.where(user_uuid: @current_user.uuid)
+      render(
+        json: InProgressFormSerializer.new(InProgressForm.where(user_uuid: @current_user.uuid)).serializable_hash
+      )
```

Comment:
switching to fast_jsonapi (which vets-api already has) made it a little easier to prevent a deep transformation of the keys